postgresqlops/postgresqlops.go

```diff
@@ -109,7 +117,7 @@ func (p *PostgreSQLOps) DeleteDatabase(name string) error {
 func (p *PostgreSQLOps) ListDatabases() ([]Database, error) {
 	dbs := []Database{}
 
-	rows, err := p.db.Query("SELECT pg_database.datname, pg_user.usename FROM pg_database, pg_user WHERE pg_database.datdba = pg_user.usesysid")
+	rows, err := p.db.Query("SELECT pg_database.datname, pg_user.usename FROM pg_database, pg_user WHERE pg_database.datdba = pg_user.usesysid AND pg_database.datname NOT IN ('postgres', 'template0', 'template1')")
```

Comment:
I added some table filtering.

Reply:
Good catch! That would cause problems if they were returned.